packages/jest-diff/src/diffStrings.ts

```diff
@@ -35,24 +26,27 @@ const bgCommon = chalk.bgYellow; // edge spaces in common line (even indentation
 const bgInverse = chalk.inverse; // edge spaces in any other lines
 
 // ONLY trailing if expected value is snapshot or multiline string.
-const highlightTrailingSpaces = (line: string, bgColor: Chalk): string =>
+const highlightTrailingSpaces = (line: string, bgColor: typeof chalk): string =>
```

Comment:
Looks like chalk exports a [TS interface](https://github.com/chalk/chalk/blob/7f6e5630b0097774d6961e9c0921697a300fbd0f/index.d.ts) too, I think we should prefer that?

Reply:
That's what I get when I do `typeof`, isn't it?

Reply:
Yes, [almost](https://github.com/chalk/chalk/blob/7f6e5630b0097774d6961e9c0921697a300fbd0f/index.d.ts#L274).
But I believe importing the interfaces from the type definitions is usually preferred, that's why they `export interface Chalk` in the first place 😄

Reply:
Cool, thanks! Updated